ksql-engine/src/main/java/io/confluent/ksql/function/UdfCompiler.java

```diff
@@ -172,7 +111,7 @@ UdfInvoker compile(final Method method, final ClassLoader loader) {
           method,
           functionName,
           description);
-      LOGGER.trace("Generated class for functionName={}, method={} class{}\n",
+      LOGGER.info("Generated class for functionName={}, method={} class\n{}\n",
```

Comment:
why the log level change? I _believe_ it already logs out UFDs at info elsewhere... (though I could be wrong). Can we ensure we're not just spamming the logs here.

Reply:
If there are lot of UDFs then this will spam the logs. That is why it was at trace

Reply:
This is another oops moment, I changed it for debugging and forgot to change it back. Thanks for the catch!